lib/compat/wordpress-6.5/block-bindings/sources/pattern.php

```diff
@@ -28,14 +28,14 @@ function gutenberg_block_bindings_pattern_overrides_callback( $source_attrs, $bl
 }
 
 function gutenberg_register_block_bindings_pattern_overrides_source() {
-	// Override the "core/pattern-attributes" source from core.
-	if ( array_key_exists( 'core/pattern-attributes', get_all_registered_block_bindings_sources() ) ) {
-		unregister_block_bindings_source( 'core/pattern-attributes' );
+	// Override the "core/pattern-overrides" source from core.
+	if ( array_key_exists( 'core/pattern-overrides', get_all_registered_block_bindings_sources() ) ) {
+		unregister_block_bindings_source( 'core/pattern-overrides' );
 	}
 	register_block_bindings_source(
-		'core/pattern-attributes',
+		'core/pattern-overrides',
 		array(
-			'label'              => __( 'Pattern Attributes' ),
+			'label'              => __( 'Pattern Overrides' ),
```

Comment:
See the related comment https://github.com/WordPress/wordpress-develop/pull/5888#discussion_r1465485730 in wordpress-develop about using [`_x`](https://developer.wordpress.org/reference/functions/_x/).

```suggestion
			'label'              => _x( 'Pattern Overrides', 'block binding source' ),
```

The same could get incorporated in other places where we use labels for this label and `Post Meta`.